packages/router/src/splash-page.tsx

```diff
@@ -1,134 +1,538 @@
+                        your CLI to get started!
+                      </p>
+                    )}
+                  </div>
+                </div>
+                <div className="resources">
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://learn.redwoodjs.com/docs/tutorial/welcome-to-redwood/"
+                      target="_blank"
+                      rel="noreferrer"
+                    ></a>
+                    <div className="icon-container">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 24 24"
+                        className="icon"
+                      >
+                        <path d="M0 0h24v24H0V0z" fill="none"></path>
+                        <path d="M4 5h16v11H4z" opacity=".3"></path>
+                        <path d="M20 18c1.1 0 1.99-.9 1.99-2L22 5c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2H0c0 1.1.9 2 2 2h20c1.1 0 2-.9 2-2h-4zM4 5h16v11H4V5zm8 14c-.55 0-1-.45-1-1s.45-1 1-1 1 .45 1 1-.45 1-1 1z"></path>
+                      </svg>
+                      <h2 className="resource-title">Tutorial</h2>
+                    </div>
+
+                    <p className="resource-description">
+                      Start here to learn how to build full-stack apps with
+                      Redwood.
+                    </p>
+                  </div>
+
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://redwoodjs.com/docs/introduction"
+                      target="_blank"
+                      rel="noreferrer"
+                    ></a>
+                    <div className="icon-container">
+                      <svg
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 24 24"
+                        className="icon"
+                      >
+                        <path d="M0 0h24v24H0V0z" fill="none"></path>
+                        <path
+                          d="M13 13l-3-2.25L7 13V4H6v16h12V4h-5z"
+                          opacity=".3"
+                        ></path>
+                        <path d="M18 2H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zM9 4h2v5l-1-.75L9 9V4zm9 16H6V4h1v9l3-2.25L13 13V4h5v16z"></path>
+                      </svg>
+                      <h2 className="resource-title">Docs and Cookbook</h2>
+                    </div>
+
+                    <p className="resource-description">
+                      Find in-depth information about Redwood features and API.
+                    </p>
+                  </div>
+
+                  <div className="resource">
+                    <a
+                      className="resource-link"
+                      href="https://community.redwoodjs.com/"
```

Comment:
Ok, so this leads to the ongoing question about _which_ community space: forums or chat? It's always confusing to offer two links. So what about this idea:
1. We create a forwarding URL like `https://redwoodjs.com/community`, which forwards to a post on the Forums
2. In the Forums post (I can create it), there will be a nice big "Welcome!" + appropriate links + intro to Forums and Chat

I like that this way would be more of a personal Welcome. And then there's space to guide/direct into joining both the Chat and Forums, and how to use each as a resource.

General thoughts?

Reply:
That sounds really welcoming. I love the idea!

Reply:
```suggestion
                      href="https://redwoodjs.com/community"
```

And done!

Reply:
I have it on my to-do list for the v0.37 release to finish the forum post.